discussions/tasks.py

```diff
@@ -44,3 +44,19 @@ def sync_discussion_users():
             api.create_or_update_discussion_user(user_id)
         except DiscussionUserSyncException:
             log.error('Impossible to sync user_id %s to discussions', user_id)
+
+
+@app.task()
+def add_contributors(channel_name, user_ids):
+    """
+    Add users to a open-discussions channel as contributors and subscribers
```

Comment:
nit: IMHO the name of the task is misleading because it sounds like it takes care only of contributors

Reply:
Right, I'll change it to `add_users_to_channel`